fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -152,6 +152,7 @@
     public static final String CBO_ENABLE_GREEDY_JOIN_REORDER = "cbo_enable_greedy_join_reorder";
     public static final String CBO_ENABLE_REPLICATED_JOIN = "cbo_enable_replicated_join";
     public static final String CBO_USE_CORRELATED_JOIN_ESTIMATE = "cbo_use_correlated_join_estimate";
+    public static final String CBO_ENABLE_LOW_CARDINALITY_OPTIMIZE = "cbo_enable_low_cardinality_optimize";
```

Comment:
This session variable seems could be removed

Reply:
We could remove this session variable after this feature is stable.